lua/neogit/status.lua

```diff
@@ -152,7 +152,22 @@ local function draw_buffer()
 
   local output = LineBuffer.new()
   if not config.values.disable_hint then
-    output:append("Hint: [<tab>] toggle diff | [s]tage | [u]nstage | [x] discard | [c]ommit | [?] more help")
+    local reversed_status_map = config.get_reversed_status_maps()
+
+    local function hint_label(map_name, hint)
+      return "[" .. reversed_status_map[map_name] .. "] " .. hint
+    end
+
+    local hints = {
+      hint_label("Toggle", "toggle diff"),
+      hint_label("Stage", "stage"),
+      hint_label("Unstage", "unstage"),
+      hint_label("Discard", "discard"),
+      hint_label("CommitPopup", "commit"),
+      hint_label("HelpPopup", "HelpPopup"),
```

Comment:
Probably should be:
```lua
hint_label("HelpPopup", "help")
```

Reply:
Oops, will fix. Thanks for catching that :sweat_smile:

Reply:
Done as part of the latest force push.